Engine/src/Engine/ResourceManagement/ShaderGenerator.h

```diff
@@ -1,6 +1,13 @@
 #pragma once
 #ifndef SHADERGENERATOR_H
 #define SHADERGENERATOR_H
+
+#include "Engine/ResourceManagement/ResourceManager.h"
```

Comment:
I don't see this class being used in this header

Reply:
It's pretty hidden, but it needs this to access GL's GLuint which is first stored here.